### DISCUSSION TITLE
Why are the Crespo Hernandez turbulence model parameters for the Gauss and GCH wake models different?

### DISCUSSION THREAD

**Discussion by clarissa-eccard** [NatLabRockies/floris#256] [Q&A]
Hello everyone,

When comparing two belt models: Gauss and GCH, I noticed that the turbulence model parameters are different in these two wake models. Was it really supposed to be like this? I had in mind that these parameters should be the same. First, I simulated the cases where there is no misalignment of the turbines and obtained a loss in the GCH model of more than 8%. However I read in the literature that for aligned turbines there is not much difference in power gain between the wake models. And for the case where the turbines are misaligned, in the GCH model, the gain in power is even greater than that of Gauss. Is correct? 
But, I got the opposite.

Thank you very much in advance

**Comment by rafmudaf** [comment]
@clarissa-eccard I'm so sorry we let this slip through the cracks! In the time since you opened this discussion, FLORIS has gone through a major redesign and nearly everything has changed. Is this question still relevant? If so, could you share a sample script?
